assembl/fabfile.py

```diff
@@ -2674,6 +2676,14 @@ def create_add_to_crontab_command(crontab_line):
     return "(crontab -l | grep -Fv '{cron}'; echo '{cron}') | crontab -".format(cron=crontab_line)
 
 
+@task
```

Comment:
is there a need for this to be a task ?

Reply:
It is to ensure that yarn is upgraded because Vincent told me that the cron is not configured on every server. It will also avoid to have different versions between developers and thus environment issues.
@vincentfretin @ayyazdaniaryan thoughts?

Reply:
Hmm, we already have `upgrade_yarn_crontab`, which is run on the main server installation path.

Reply:
@ayyazdaniaryan on existing servers, the cron is not installed everywhere. On staging: 
```
yarn -v
1.6.0
```
is not the last version.